extensions/amp-form/0.1/amp-form.js

```diff
@@ -216,6 +221,29 @@ export class AmpForm {
   }
 
   /**
+   * Returns form data as an object.
+   * @return {!Object}
+   * @private
+   */
+  getFormAsObject_() {
+    const data = {};
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
+    for (let i = 0; i < inputs.length; i++) {
+      const input = inputs[i];
+      if (!input.name || !input.value) {
```

Comment:
`disabled` inputs should also be excluded.

(checking `.disabled` won't be enough since it won't cover cases where one or more ancestor `fieldset` is disabled, so you need to walk up the tree to find all `fieldset` ancestors in the form and check if any of them are disabled)